lib/fungus_toast/games/grid_cell.ex

```diff
@@ -1,3 +1,3 @@
 defmodule FungusToast.Games.GridCell do
-  defstruct index: nil, live: false, empty: true, player_id: nil, previous_player_id: nil
+  defstruct index: nil, live: false, empty: true, out_of_grid: false, player_id: nil, previous_player_id: nil
```

Comment:
Why would a GridCell ever be `out_of_grid`? The name implies that it's a cell in the grid

Reply:
In %SurroundingCells{} we'll want to know if a given adjacent slot is off the grid (i.e. this cell is on the crust). There are numerous ways to handle this, but i'm just mimicking how I did it in C# since I know it works. No doubt there is a better way :)